sections/announcement-bar.liquid

```diff
@@ -2,11 +2,12 @@
   {%- case block.type -%}
     {%- when 'announcement' -%}
       <div class="announcement-bar color-{{ block.settings.color_scheme }} gradient" role="region" aria-label="{{ 'sections.header.announcement' | t }}" {{ block.shopify_attributes }}>
+        <div class="page-width">
```

Comment:
We should adjust some of the spacing/indentation from `{%- if block.settings.text != blank -%}`:
- [Lines 6-19](https://github.com/Shopify/dawn/blob/e14c33fe6f1b4ecbc37233dcea0a3dc529417b0a/sections/announcement-bar.liquid#L6-L19) ideally should be changed **from**:

```liquid
        <div class="page-width">
        {%- if block.settings.text != blank -%}
          {%- if block.settings.link != blank -%}
            <a href="{{ block.settings.link }}" class="announcement-bar__link link link--text focus-inset animate-arrow">
          {%- endif -%}
              <p class="announcement-bar__message {{ block.settings.text_alignment }} h5">
                {{ block.settings.text | escape }}
                {%- if block.settings.link != blank -%}
                  {% render 'icon-arrow' %}
                {%- endif -%}
              </p>
          {%- if block.settings.link != blank -%}
            </a>
          {%- endif -%}
        {%- endif -%}
        </div>
```

To this:

```liquid
        <div class="page-width">
          {%- if block.settings.text != blank -%}
            {%- if block.settings.link != blank -%}
              <a href="{{ block.settings.link }}" class="announcement-bar__link link link--text focus-inset animate-arrow">
            {%- endif -%}
            <p class="announcement-bar__message {{ block.settings.text_alignment }} h5">
              {{ block.settings.text | escape }}
              {%- if block.settings.link != blank -%}
                {% render 'icon-arrow' %}
              {%- endif -%}
            </p>
            {%- if block.settings.link != blank -%}
              </a>
            {%- endif -%}
          {%- endif -%}
        </div>
```